src/Extensions/PdfExportControllerExtension.php

```diff
@@ -90,6 +91,9 @@ public function generatePDF()
         }
 
         $binaryPath = $this->owner->data()->config()->get('wkhtmltopdf_binary');
+        if ($binaryPath) {
+            Deprecation::notice('3.0', 'Using wkhtmltopdf_binary config is deprecated. Define the path using WKHTMLTOPDF_BINARY env var instead.');
```

Comment:
Mind fixing this line length? We had a 120 char hard limit

Reply:
On it!

Reply:
All good now @robbieaverill?